src/OrbitGl/CaptureWindow.cpp

```diff
@@ -725,8 +727,11 @@ void CaptureWindow::RenderSelectionOverlay() {
   uint64_t max_time = std::max(select_start_time_, select_stop_time_);
 
   float from_world = time_graph_->GetWorldFromTick(min_time);
+  from_world = time_graph_->ClampToTimelineUiElementWorldX(from_world);
   float to_world = time_graph_->GetWorldFromTick(max_time);
+  to_world = time_graph_->ClampToTimelineUiElementWorldX(to_world);
   float stop_pos_world = time_graph_->GetWorldFromTick(select_stop_time_);
+  stop_pos_world = time_graph_->ClampToTimelineUiElementWorldX(stop_pos_world);
```

Comment:
I guess the old calls can now be removed?

Reply:
well those are still used here, right?

Reply:
Ohhh, I get it now. Sorry never mind.

You can consider writing it in a single expression (or not - what you deem is best).